tests/udp/test_raw_traffic_with_fixed_udp_ports.py

```diff
@@ -0,0 +1,78 @@
+from abstract_open_traffic_generator.flow import *
+import pytest
+
+from abstract_open_traffic_generator import flow
+import utils
+
+
+@pytest.mark.parametrize('packets', [1000])
+@pytest.mark.parametrize('size', [74, 128, 1518])
+def test_raw_traffic_with_fixed_udp_ports(api, b2b_raw_config, size, packets):
+    """
+    Configure a raw udp flow with,
+    - fixed src and dst Port address
+    - 1000 frames of 74B,128B,1518B size each
+    - 10% line rate
+
+    Validate,
+    - tx/rx frame count is as expected
+    - all captured frames have expected src and dst Port address
+    """
+    f = b2b_raw_config.flows[0]
+
+    f.packet = [
+        flow.Header(
+            flow.Ethernet(
+                src=flow.Pattern('00:0c:29:1d:10:67'),
+                dst=flow.Pattern('00:0c:29:1d:10:71')
+                )
+            ),
+        flow.Header(
+            flow.Ipv4(
+                src=flow.Pattern("10.10.10.1"),
+                dst=flow.Pattern("10.10.10.2")
+                )
+            ),
+        flow.Header(
+            flow.Udp(
+                src_port=flow.Pattern("6"),
```

Comment:
This is not really a technical issue, but let's use ports outside 1-1024 range (something that will require more than 1 byte).

Reply:
updated